server/service/src/sync/translations/requisition.rs

```diff
@@ -432,10 +432,13 @@ impl SyncTranslation for RequisitionTranslation {
 fn from_legacy_sent_datetime(
     last_modified_at: i64,
     r#type: &RequisitionType,
+    status: &LegacyRequisitionStatus,
 ) -> Option<NaiveDateTime> {
     match r#type {
         RequisitionType::Request => {
-            if last_modified_at > 0 {
+            // In OG, a finalised "fn" request requisition is the equivalent of a "sent" request requisition in OMS.
+            // There are no date/time fields in OG requisition table for this, there are logs though. Hence using last_modified_at.
+            if last_modified_at > 0 && matches!(status, LegacyRequisitionStatus::Fn) {
```

Comment:
I considered just using regular bool expression but this is nice if we for some reason wanted to include statuses?

Reply:
Was exactly what I was thinking /o/